### PR TITLE
Add audio duration filter clarifications as strings

### LINKED ISSUE #1538: Add clarification for length filters
## Problem

It can be difficult to understand what the length filter names mean.

## Description

We should add clarification. Something like:
- [ ] shortest under 30 seconds
- [ ] short 30 seconds - 2 minutes
- [ ] medium 2 - 10 minutes
- [ ] long above 10 minutes
Before implementing this, we need a design for it. This is what the current filters look like:
Screen Shot 2022-06-19 at 7 11 18 AM

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #1538 by @obulat

## Description

I've added the clarification as text so this PR does not change anything in the code. But if we want to style them differently, I would need to separate the names from the clarification texts. Please, let me think what you think.

Audio length filters with clarification

## Testing Instructions

Run the app and open the Audio search page filters.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -333,10 +333,10 @@
     },
     "lengths": {
       "title": "Durations",
-      "shortest": "Shortest",
-      "short": "Short",
-      "medium": "Medium",
-      "long": "Long"
+      "shortest": "Shortest (< 30s)",
```
Comment (zackkrida) [inline src/locales/en.json:336]:
```suggestion
      "shortest": "Shortest (<30s)",
```

For consistency with the 'long' label

File: src/locales/en.json [hunk 2]
```diff
@@ -333,10 +333,10 @@
...
+      "short": "Short (30s - 2min)",
+      "medium": "Medium (2 - 10min)",
```
Comment (zackkrida) [inline src/locales/en.json:338]:
```suggestion
      "short": "Short (30s-2min)",
      "medium": "Medium (2-10min)",
```

I believe with these kinds of ranges there isn't generally a space.

**Comment by panchovm** [comment]
It looks simple and works great.

My only suggestion is to eliminate the classification (shortest, short, medium, and long) and keep the time length, as SoundCloud does.

CleanShot 2022-06-29 at 09 53 51@2x

On the other hand, I wonder if it is fine using abbreviations such as `min` for meeting i18n requirements. If so, then I suggest using `sec` for seconds and adding a whitespace character between the number and the word. The outcome would be:

```
< 30 sec
30 sec-2 min
2-10 min
> 10 min
```

**Comment by zackkrida** [comment]
I decided to ask about this in the #polyglots channel. We can see if they have any suggestions:

https://wordpress.slack.com/archives/C02RP50LK/p1656514587496599

**Comment by obulat** [comment]
> My only suggestion is to eliminate the classification (shortest, short, medium, and long) and keep the time length, as SoundCloud does.

That is actually much simpler and easier to understand ! :)

Thank you for asking in the polyglot channel, @zackkrida ! I think that these translations should be simple because they are hard-coded, and the values for seconds or minutes in strings like '30 sec - 1 min' do not change. If they did, we would have to go the pluralization route, which usually requires more effort. I'm going to go look at the channel :)

**Comment by obulat** [comment]
Updated screenshot:

Screen Shot 2022-06-30 at 5 31 54 PM

File: src/locales/en.json [hunk 3]
```diff
@@ -333,10 +333,10 @@
     },
     "lengths": {
       "title": "Durations",
```
Comment (panchovm) [inline src/locales/en.json:335]: For consistency, I think we should use `Duration`, without the `s`. All the other categories are written in a singular form.

Comment (obulat) [reply src/locales/en.json:335]: Great catch, @panchovm !

Comment (obulat) [inline src/locales/en.json:335]:
```suggestion
      "title": "Duration",
```

**Review by zackkrida — APPROVED** [review]

**Review by panchovm — APPROVED** [review]
It looks great ✨

**Review by zackkrida — APPROVED** [review]

**Pull request merged**